python/lsst/analysis/tools/actions/vector/selectors.py

```diff
@@ -494,3 +494,81 @@ def __call__(self, data: KeyedData, **kwargs) -> Vector:
             # No band selection is applied, i.e., select all rows
             mask = np.full(len(data[self.vectorKey]), True)  # type: ignore
         return cast(Vector, mask)
+
+
+class ParentObjectSelector(FlagSelector):
```

Comment:
Yeah, I think we wrote these during pair coding that one day just to get something running. But now that I understand how the package works, I got rid of `PrimarySelector` (since it is only a `FlagSelector` with `selectWhenTrue = ["detect_isPrimary"]`) and simplified `ParentObjectSelector` to inherit from FlagSelector and only implement the `setDefaults` method.